### DISCUSSION TITLE
[Plugin] Support for Xamarin.Community.Toolkit Popups

### DISCUSSION THREAD

**Discussion by chowarth** [PrismLibrary/Prism#2390] [Xamarin.Forms] [answered]
Hi,

I watched the recent video on the new Xamarin.Community.Toolkit popups and started wondering about how it would be possible to use them with Prism, so I've put together a PoC [here](https://github.com/chowarth/Prism.XCT.Popups) - it basically follows the same pattern as the `IDialogService`.

I imagine it wouldn't be something that would live directly in the Prism project but separately.

Is this something that you would be open to being created?

**Comment by dansiegel** [comment, marked as answer]
As the one behind Popup's I'm sure @ahoefling can comment more on this... but as I understand it, he already has a working plugin and was planning on releasing it in the near future.

**Reply by chowarth** [reply]
Ah OK. I'll wait and see what he has in store then 😃

**Reply by chowarth** [reply]
@dansiegel would you be OK with me putting this out as a temporary solution while awaiting a more official package? With a name change of course.

**Reply by SkyeHoefling** [reply]
I'm on paternity leave and won't be back until June. Let me check with work to see if I can get some priority to release the code.

I am really excited about releasing the plugin, I just never imagined so many people would want it. The code is tightly coupled to our proprietary code base so I need to extract and create the repo.

**Reply by chowarth** [reply]
No worries, congratulations and enjoy this time!

**Reply by SkyeHoefling** [reply]
@chowarth I have published our code for the plugin to GitHub. I will be publishing it to NuGet once the next version of XCT is released as it has some bits that we need. If you clone the repo you can see the plugin is working with Prism 8.1.
* https://github.com/FileOnQ/Prism.Popups.XCT
